maps/turtlebot3_waffle_pi/map.yaml

```diff
@@ -1,6 +1,6 @@
 image: map.pgm
 resolution: 0.050000
-origin: [-10.000000, -10.000000, 0.000000]
+origin: [-12.500000, -12.500000, 0.000000]
```

Comment:
How is it calculated?

Reply:
It's a new map.. the origin comes from the map config used for the map generation (https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/ros1/map_config/small_house.rb) 

Reference: https://github.com/marinaKollmitz/gazebo_ros_2Dmap_plugin/blob/0820610f46235cd7ce1458ea030ef83b1616da37/src/gazebo_2Dmap_plugin.cpp#L313

Reply:
Got it, thanks